modules/lib/src/main/resources/assets/js/app/wizard/ContentWizardPanel.ts

```diff
@@ -1234,7 +1229,24 @@ export class ContentWizardPanel
         this.settingsWizardStep = new ContentWizardStep(i18n('field.settings'), this.settingsWizardStepForm, 'icon-wrench');
         steps.push(this.settingsWizardStep);
 
-        return steps;
+        const editPermissionsIcon = this.canEveryoneRead(this.getPersistedItem()) ? 'icon-unlock' : 'icon-lock';
+        this.editPermissionsToolbarButton = new ContentWizardStep(i18n('field.access'), this.settingsWizardStepForm, editPermissionsIcon);
+        this.editPermissionsToolbarButton.getTabBarItem().addClass('edit-permissions-button');
+        this.editPermissionsToolbarButton.getTabBarItem().onClicked(this.handleEditPermissionsButtonClicked.bind(this));
+        steps.push(this.editPermissionsToolbarButton);
+
+        return this.addAccessibilityToSteps(steps);
```

Comment:
You're not actually creating some new steps, so it is better to add accessibility separately to avoid confusion
```
this.addAccessibilityToSteps(steps);
return steps;
```